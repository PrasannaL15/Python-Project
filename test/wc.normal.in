This is a text with various special characters, such as @, #, $, %, and ^. 
It also includes numbers 123
which might be treated differently by the word count utility.
