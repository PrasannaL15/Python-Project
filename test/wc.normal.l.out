       3 test\wc.normal.in
